Fundamentals/Design/Stylesheets-Javascript/index-v9.md

```diff
@@ -0,0 +1,148 @@
+verified-against: rc-02
+state: complete
+updated-links: true
+---
+
+# Working with stylesheets and JavaScript
+
+## Stylesheets in the Backoffice
+
+You can create and edit stylesheets in the Stylesheets folder in the Settings section of the Backoffice.
+
+![Creating a new stylesheet](images/1-creating-stylesheet.png)
+
+In the Create menu, there are several options available:
+
+* Stylesheet file (for use in templates/views)
+* Rich Text Editor stylesheet file (for use in [Rich Text Editor](../../Backoffice/Property-Editors/Built-in-Property-Editors/Rich-Text-Editor/index-v9.md))
+* Folder (for keeping stylesheets organized)
+
+:::note
+It is currently not possible to use any CSS preprocessor (such as SASS) in the backoffice.
+:::
+
+After creating a new stylesheet, you would work with it as you would with templates or javascript files - using the built-in backoffice text editor.
+When you're working with stylesheets, you also have access to the Rich Text Editor, which allows you to create CSS styles and get a real-time preview.
+
+![Stylesheet RTE](images/2-rte-editor.png)
+
+The rules you create in the Rich Text Editor section will carry over to the Code tab.
+
+![Stylesheet RTE tab](images/3-rte-editor-p2.png)
+![Stylesheet Code tab](images/3-rte-editor-p3.png)
+
+To reference your newly included stylesheet in a template file, navigate to Templates, pick the template you like (css files are usually referenced in the layout or home templates) and link to it with the `link` tag.
+
+![Linking CSS in template](images/4-link-css-v9.png)
+
+By default, the stylesheets will be saved in the `wwwroot/css` folder in the solution.
+To reference them you can use either of the methods used in the above screenshot.
+
+```html
+<link rel="stylesheet" href='@Url.Content("~/css/mystylesheet.css")' />
+```
+or
+```html
+<link rel="stylesheet" href="/css/mystylesheet.css" />
+```
+
+With the stylesheet referenced, you will be able to style the template file with the rules and classes defined in the stylesheet.
+
+Your stylesheets can be used in Rich Text Editors (datatype) as well - please see the [Rich Text Editor](../../Backoffice/Property-Editors/Built-in-Property-Editors/Rich-Text-Editor/RTE-Styles/index.md) documentation for more information.
+
+:::note
+If your RTE is styled differently on the frontend of the site, the backoffice styling might be getting overwritten by other stylesheets you have included.
+:::
+
+## JavaScript files in the Backoffice
+
+To create and edit JavaScript files in the Backoffice, head on over to the Scripts folder in the Settings section of the Backoffice.
+
+![Creating a new JavaScript](images/8-create-js.png)
+
+From here you can add a new JavaScript file, or a new folder.
+
+Add a new JavaScript file and write your code:
+
+![Sample JS script](images/9-myscript.png)
+
+Then, navigate to the template where you would like to include your JS file.
+
+```html
+<script src="/scripts/myScript.js"></script>
```

Comment:
@nikolajlauridsen Just looked it up and it is because with HTML v5 page does not require the type attribute because the default script language is 'text/javascript' in a <script> tag 🙂

Reply:
Oh neat, didn't know that, the more you know \*insert shooting star gif*